plugins/shared-react/src/utils/date/format-date.ts

```diff
@@ -1,9 +1,29 @@
+/* export function formatDate(date: string | number | Date) {
+  if (date === -1) {
+    return 'N/A';
+  }
+
+  const adjustedDate = typeof date === 'number' ? date * 1000 : date;
+  return new Date(adjustedDate).toLocaleString('en-US', {
+    timeZone: new Intl.DateTimeFormat().resolvedOptions().timeZone,
+    timeStyle: 'short',
+    dateStyle: 'medium',
+  });
+} */
```

Comment:
Please remove commented out code

Reply:
@tumido Resolved in add46e92718a8dc435d3eeae04a282fd26a90ffa